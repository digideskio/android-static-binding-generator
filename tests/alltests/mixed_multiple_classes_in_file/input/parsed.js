var MyClass = (function (_super) {
    __extends(MyClass, _super);
    function MyClass() {
        _super.apply(this, arguments);
    }
    MyClass.prototype.MyClassMethod111 = function () {
    };
    MyClass.prototype.MyClassMethod1111 = function () {
    };
    return MyClass;
})(android.view.ViewGroup1111);
var MyButton = android.widget.Button22222222.extend("ClassFromOrigina2222222222", {
	onClick222222: function () {
	},
	onClick222222222: function () {
	}
});
var MyClass = (function (_super) {
    __extends(MyClass, _super);
    function MyClass() {
        _super.apply(this, arguments);
    }
    MyClass.prototype.MyClassMethod3333 = function () {
    };
    MyClass.prototype.MyClassMethod33333333 = function () {
    };
    MyClass = __decorate([
        JavaProxy("asd.asd.As3333")
    ], MyClass);
    return MyClass;
})(android.view.ViewGroup333333);
(function () {
    var MyButton = android.widget.Button44444444.extend("my.custom.ClassFromOrigina44444", {
        onClick4444: function () {
        },
        onClick4444444: function () {
        }
    });
})();
