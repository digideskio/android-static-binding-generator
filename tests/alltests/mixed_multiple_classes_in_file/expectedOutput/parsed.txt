android.view.ViewGroup333333**MyClassMethod3333,MyClassMethod33333333*asd.asd.As3333
android.widget.Button44444444**onClick4444,onClick4444444*my.custom.ClassFromOrigina44444
android.view.ViewGroup1111*_frnal_prepareExtend_l62_c37__MyClass*MyClassMethod111,MyClassMethod1111*
android.widget.Button22222222*_fparsed_l12_c46__ClassFromOrigina2222222222*onClick222222,onClick222222222*